socorro/external/fs/crashstorage.py

```diff
@@ -28,10 +28,11 @@
 from socorro.lib.util import DotDict
 
 
-def dates_to_strings_for_json(obj):
-    if isinstance(obj, datetime.datetime):
-        return obj.isoformat()
-    return json.JSONEncoder.default(self, obj)
+class JSONISOEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, datetime.date):
+            return obj.isoformat()
+        raise NotImplementedError("Don't know about {0!r}".format(obj))
```

Comment:
This was taken from the boto crashstorage code.

We should probably centralize this in lib, but I decided it's easier to copy it. It replaces a function that uses "self", but really shouldn't.